src/ccache.c

```diff
@@ -3470,7 +3472,7 @@ ccache_main_options(int argc, char *argv[])
 	};
 
 	int c;
-	while ((c = getopt_long(argc, argv, "cChF:M:o:psVz", options, NULL)) != -1) {
+	while ((c = getopt_long(argc, argv, "cChF:M:k:o:psVz", options, NULL)) != -1) {
```

Comment:
Please keep the option list in sorted order. Also, it would be nice to keep the line [a bit narrower](https://github.com/ccache/ccache/blob/master/CONTRIBUTING.md#code-style).

Reply:
I cut the line before `!= 1`.